vioscsi/helper.h

```diff
@@ -49,6 +49,8 @@
 #define CACHE_LINE_SIZE 64
 #define ROUND_TO_CACHE_LINES(Size)  (((ULONG_PTR)(Size) + CACHE_LINE_SIZE - 1) & ~(CACHE_LINE_SIZE - 1))
 
+#define VQ_ADD_BUFFER_SUCCESS 0
```

Comment:
Can you have this be the value that gets returned too? That would help glue this all together, so if you did a git grep in the future you would see it super crisply

Reply:
I was trying to avoid touching anything VirtIO (in this case  in `VirtIO\VirtIORing.c` and `VirtIO\VirtIORing-Packed.c`)... Is that the MO for the repo? It seemed to me we avoid making changes there if possible.

Reply:
I guess if I was splitting hairs, I'd have this in two commits. First being adding that to virtio common code in general, and a second commit to plumb into scsi code. Just my 2 cents

Reply:
Thanks Jon. I've added a second commit to add the definition to VirtIO instead and updated the relevant VirtIO routines to use the new definition.